src/Cascader/Cascader.tsx

```diff
@@ -66,6 +66,9 @@ export interface CascaderProps<T = ValueType>
   /** Custom render menu items */
   renderMenuItem?: (itemLabel: React.ReactNode, item: ItemDataType) => React.ReactNode;
 
+  /** Custom render search items */
+  renderSearchItem?: (itemLabel: React.ReactNode, item: ItemDataType) => React.ReactNode;
+
```

Comment:
I'd suggest passing the complete parent-child path of the search result as arguments for flexibility.
```suggestion
  /** Custom render search items */
  renderSearchItem?: (itemLabel: React.ReactNode, items: ItemDataType[]) => React.ReactNode;

```